src/containerapp-compose/HISTORY.rst

```diff
@@ -5,4 +5,9 @@ Release History
 
 0.1.0
 ++++++
-* Initial release.
\ No newline at end of file
+* Initial release.
+
+0.2.0
+++++++
+* Add basic build support
+* Fix bug in specifying memory
```

Comment:
Please write the description of the new version at the top of the history note

Reply:
I'm not exactly sure what you mean, but I tried to clarify the first item which is the new feature description.

Reply:
Please refer to [HISTORY.md](https://github.com/Azure/azure-cli-extensions/blob/main/src/spring/HISTORY.md) to write the description of the new version above the description of the old version

Reply:
Now I see @zhoxing-ms .  Thank you, I misunderstood the direction.  I made the adjustment.